zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -147,6 +154,19 @@ private Set<String> getRoleAuthorizedResourceIdentifiers(
         .collect(Collectors.toSet());
   }
 
+  private Set<String> getGroupAuthorizedResourceIdentifiers(
+      final List<Long> groupKeys,
+      final AuthorizationResourceType resourceType,
+      final PermissionType permissionType) {
+    return groupKeys.stream()
+        .flatMap(
+            groupKey ->
+                authorizationState
+                    .getResourceIdentifiers(groupKey, resourceType, permissionType)
+                    .stream())
+        .collect(Collectors.toSet());
+  }
```

Comment:
💭 🔧  I suppose we don't even need a specific group one. This method is an exact dupe of the `getRoleAuthorizedResourceIdentifiers` method. We could combine them in a single generic method.